app/index.html

```diff
@@ -34,45 +34,23 @@
 
   </head>
   <body id="app">
+
     <!--[if lt IE 10]>
         <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
+    <![endif]-->
+
+    <!-- ==============================   APP   ============================== -->
 
     <!-- black bar on the top -->
-    <nav class="navbar navbar-default navbar-static-top navbar-inverse" id="top-bar">
-      <div class="container-fluid">
-        <div class="navbar-header">
-          <!-- Title -->
-          <a href="/" class="navbar-brand">
-            <img src="images/logo/delft3d-gt.svg" alt="logo"/>
-          </a>
-        </div>
-        <div class="collapse navbar-collapse">
-          <!-- Items -->
-          <ul class="navbar-nav nav">
-            <li><a href="#"><h3>Delft3D<br /><small>Geological Tool</small></h3></a></li>
-          </ul>
-          <ul class="navbar-nav nav navbar-right">
-            <li><user-details></user-details></li>
-            <li><a class="btn btn-primary navbar-btn" title="Create scenario" href="#/scenarios/create">Create scenario</a></li>
-            <li><a class="btn btn-primary navbar-btn" href="/api/v1/" title="rest api documentation (for developers)"><span class="glyphicon glyphicon-list-alt" aria-hidden="true"></span></a></li>
-            <li><a class="btn btn-primary navbar-btn" href="/logout/">Logout</a></li>
-          </ul>
-
-        </div>
-      </div>
-    </nav>
-
-
-    <div  class="full-height">
-      <!-- This will be filled by app.js (vue-router) -->
-      <router-view keep-alive></router-view>
-      <!-- and no footer. -->
-    </div>
+    <navbar></navbar>
```

Comment:
web component convention is to use two words (https://www.w3.org/TR/custom-elements/#valid-custom-element-name)